echopype/tests/convert/test_convert.py

```diff
@@ -114,43 +281,18 @@ def test_convert_time_encodings(sonar_model, raw_file, xml_path):
     os.unlink(ed.converted_raw_path)
 
 
-@pytest.mark.parametrize("model", ["EK60"])
-@pytest.mark.parametrize(
-    "input_path",
-    [
-        "./echopype/test_data/ek60/ncei-wcsd/Summer2017-D20170615-T190214.raw",
-        "s3://data/ek60/ncei-wcsd/Summer2017-D20170615-T190214.raw",
-        [
-            "http://localhost:8080/data/ek60/ncei-wcsd/Summer2017-D20170615-T190214.raw",
-            "http://localhost:8080/data/ek60/ncei-wcsd/Summer2017-D20170615-T190843.raw",
-        ],
-    ],
-)
-@pytest.mark.parametrize("export_engine", ["zarr", "netcdf4"])
-@pytest.mark.parametrize(
-    "output_save_path",
-    [
-        None,
-        "./echopype/test_data/dump/",
-        "./echopype/test_data/dump/tmp.zarr",
-        "./echopype/test_data/dump/tmp.nc",
-        "s3://ooi-raw-data/dump/",
-        "s3://ooi-raw-data/dump/tmp.zarr",
-        "s3://ooi-raw-data/dump/tmp.nc",
-    ],
-)
 def test_convert_ek60(
-    model,
-    input_path,
+    ek60_input_paths,
     export_engine,
     output_save_path,
     minio_bucket,
+    model="EK60",
 ):
```

Comment:
Not trying to make this over complicated, but the content of `test_convert_*` (* is an allowable sonar_model) is identical except for the AZFP case where an XML file is needed. It seems that these can be consolidated also using the power of fixtures so later changes are easier to maintain?